### ISSUE TITLE
no way to take input

### ISSUE THREAD

**Issue by TheSinding** [Microsoft/vscode-java-debug#86]
Hi, is it me or is there no way to take input from the user, with ex.
```java
public class App {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        System.out.println("Hello " + s.nextLine());
    }
}
```

There is no console to type input into ? 
I've tried using the "console": "internalTerminal" and other's but it does not work with any of that, is it an limitation from this ext ?

**Comment by testforstephen** [comment]
@TheSinding currently the debug console doesn't support typing input yet.
But you can launch your java program manually in a standalone terminal.
`java -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=1049 -cp ./bin YourMainClass`

Then start the debug with attach mode.
<pre>
    {
      "type": "java",
      "name": "Java Debug (Attach)",
      "request": "attach",
      "hostName": "localhost",
      "port": 1049
   }
</pre>

**Comment by TheSinding** [comment]
Arh okay, thanks!

**Comment by akaroml** [comment]
@testforstephen can we support launching in the integrated terminal for this scenario?

**Comment by testforstephen** [comment]
@akaroml yes, the feature `supportsRunInTerminalRequest` provided by VSCode is exactly for this scenario.

**Comment by testforstephen** [comment]
It's fixed in 0.4.0. By changing `console` to `integratedTerminal/externalTerminal` in launch.json, it supports console input now.

**Comment by tuanvpham** [comment]
@testforstephen thanks man.
